addons/stock/models/stock_move.py

```diff
@@ -234,7 +234,7 @@ def _compute_has_move_lines(self):
             move.has_move_lines = bool(move.move_line_ids)
 
     @api.one
-    @api.depends('product_id', 'product_uom', 'product_uom_qty')
+    @api.depends('product_id', 'product_uom', 'product_uom.factor', 'product_uom_qty')
```

Comment:
Not a good idea to recompute values of existing entries. It's like recomputing unit prices on SO/PO/Invoice lines automatically... So no, don't do that.